src/rules/default.js

```diff
@@ -1,8 +1,12 @@
 import Exports from '../ExportMap'
 
+const ruleDocsUrl = 'https://github.com/benmosher/eslint-plugin-import/tree/master/docs/rules'
```

Comment:
Can we import this from somewhere rather than repeating it everywhere?

Reply:
Maybe something like `docsUrl('rule-name')`?

Reply:
Yep, can do that for everything other than `first`, since it needs a specific blob.

I didn't really like how this was fitting in the style here which is why I explicitly called it out as something to look at 😉.